pkg/auth/server.go

```diff
@@ -65,6 +66,8 @@ func NewServer(ctx context.Context, cfg *rest.Config) (*Server, error) {
 		return nil, err
 	}
 
+	sc.Wrangler = wContext
```

Comment:
I might be missing something, but what is this for? I don't see where we make use of it in this PR

Reply:
In some places we pull wrangler out of a scaled context to use the clients.
At least it was the case with my code. I have not checked yet if the rewritten code does the same.
Regardless, during Rancher init we had a number of scaled contexts which did not have wrangler set,
and my code ran into this nil pointer.
I fixed this for only the places which reached my code.

I believe that Peter has now fixed this for the other places as well, to ensure proper init everywhere.